src/components/OnBoarding/ThirdForm.jsx

```diff
@@ -0,0 +1,316 @@
+import { useState } from 'react';
+import propTypes from 'prop-types';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Button,
+  Checkbox,
+  Text,
+  Box,
+  Flex,
+  FormLabel,
+  FormControl,
+  Input,
+  Stack,
+  Select,
+  Alert,
+  AlertIcon,
+  Link,
+  AlertTitle,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverHeader,
+  PopoverBody,
+  PopoverArrow,
+  PopoverCloseButton,
+  HStack,
+  Image,
+} from '@chakra-ui/react';
+import FPH_LOGO from './fph_logo.png';
+
+const ThirdForm = ({ handleChange, prevStep, handleSubmit }) => {
+  const daysOfWeek = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+  const [businessHours, changeBusinessHours] = useState(
+    daysOfWeek.reduce(
+      (acc, day) => ({
+        ...acc,
+        [day]: {
+          start: { hour: '9', minute: '00', ampm: 'AM' },
+          end: { hour: '5', minute: '00', ampm: 'PM' },
+        },
+      }),
+      {},
+    ),
+  );
```

Comment:
dont create these states, use the states from `formData` in here, such that when we press the back button, we don't lose all of our entries